source/containers.tex

```diff
@@ -6996,7 +6996,7 @@
 of interest, but in a \tcode{forward_list} there is no constant-time way to access a
 preceding element.
 For this reason, \tcode{erase_after} and \tcode{splice_after}
-take fully-open ranges, not semi-open ranges.
```

Comment:
This is an interesting case: I understand that, in isolation, an adverb qualifies an adjective. But here we're drawing from a coherent family of terms like "half-open", "semi-open", "closed-open", "open-open", "fully-open", "fully-closed" (some of which are synonyms), where there isn't always an adverb as the first part, but it's still useful to have a common style to show that these are all members of the same technical domain, and we don't mean "fully open" in some other, broader sense. ("You've not been fully open about your use of fully-open ranges")

(Not to be confused with "clopen", which is both open and closed at once, and [something different](https://en.wikipedia.org/wiki/Clopen_set) :-) )